framework/tlvf/src/include/tlvf/ClassList.h

```diff
@@ -11,10 +11,10 @@
 #define _ClassList_H_
 
 #include <memory>
-#include <tlvf/BaseClass.h>
-#include <tlvf/ieee_1905_1/eTlvType.h>
 #include <vector>
 
+class BaseClass;
```

Comment:
There's no need for a forward declaration here.

Reply:
How not? we are declaring multiple functions which return shared_ptr to BaseClass, this is needed.